versioneer.py

```diff
@@ -1244,7 +1253,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, runner=run_command):
     # if there is a tag matching tag_prefix, this yields TAG-NUM-gHEX[-dirty]
     # if there isn't one, this yields HEX[-dirty] (no NUM)
     describe_out, rc = runner(GITS, [
-        "describe", "--tags", "--dirty=", "--always", "--long",
+        "describe", "--tags", "--dirty", "--always", "--long",
```

Comment:
This is probably wrong. We dirty the repo by adding license info before builds without committing it, which makes it dirty, but we don't want dirty to show up in the version. This is one reason we never upgraded versioneer, we needed to modify it.

Long term, we probably want to drop versioneer, it is over designed for our needs.

Reply:
I see. Re-applied 2145d9e.